src/Bloc/BlElement.class.st

```diff
@@ -3423,13 +3430,64 @@ BlElement >> spaceDo: aBlock [
 	self space ifNotNil: aBlock
 ]
 
+{ #category : #'text input' }
+BlElement >> startTextInput [
+
+	"Used by a morph to request text edition capabilities.
+	The rendering backend can use this information to, for example, open the IME (input-method-editor). This version or requestTextEditingAt: **must** be sent before any text input event can be received.
+	Typically, when an editor get the focus. requestTextEditingAt: manages the IME (input-method-editor) opening position.
+	
+	Use it in pair with requestStopTextEditing
+	
+	propagate the request to the space."
+	spaceReference startTextInput
```

Comment:
I think it could be `spaceDo: [ :space | space startTextInput ]` to follow the pattern of other API methods. 
But not so grave. Wha tdo you think?